solana-programs/anchor/audius-data/tests/audius-data.ts

```diff
@@ -1115,8 +1027,158 @@ describe("audius-data", function () {
+        userAuthorityKeypair: delegate.userAuthorityDelegateKeypair, // substitute delegate
+        trackOwnerPDA: delegate.userAccountPDA,
+        userAuthorityDelegateAccountPDA: SystemProgram.programId, // missing PDA
+        authorityDelegationStatusAccountPDA:
+          delegate.authorityDelegationStatusPDA,
+      })
+    ).to.be.rejectedWith(Error);
+
+    await expect(
+      testCreateTrack({
+        provider,
+        program,
+        id: trackID,
+        baseAuthorityAccount: delegate.baseAuthorityAccount,
+        handleBytesArray: delegate.handleBytesArray,
+        adminStorageAccount: adminStorageKeypair.publicKey,
+        bumpSeed: delegate.userBumpSeed,
+        trackMetadata,
+        userAuthorityKeypair: delegate.userAuthorityDelegateKeypair, // substitute delegate
+        trackOwnerPDA: delegate.userAccountPDA,
+        userAuthorityDelegateAccountPDA: delegate.userAuthorityDelegatePDA,
+        authorityDelegationStatusAccountPDA: SystemProgram.programId, // missing PDA
+      })
+    ).to.be.rejectedWith(Error);
+
+    await expect(
+      testCreateTrack({
+        provider,
+        program: anchor.web3.Keypair.generate().publicKey, // wrong program
+        id: trackID,
+        baseAuthorityAccount: delegate.baseAuthorityAccount,
+        handleBytesArray: delegate.handleBytesArray,
+        adminStorageAccount: adminStorageKeypair.publicKey,
+        bumpSeed: delegate.userBumpSeed,
+        trackMetadata,
+        userAuthorityKeypair: delegate.userAuthorityDelegateKeypair, // substitute delegate
+        trackOwnerPDA: delegate.userAccountPDA,
+        userAuthorityDelegateAccountPDA: delegate.userAuthorityDelegatePDA,
+        authorityDelegationStatusAccountPDA:
+          delegate.authorityDelegationStatusPDA,
+      })
+    ).to.be.rejectedWith(Error);
+
+    await expect(
+      testCreateTrack({
+        provider,
+        program: anchor.web3.Keypair.generate().publicKey, // wrong program
+        id: trackID,
+        baseAuthorityAccount: delegate.baseAuthorityAccount,
+        handleBytesArray: delegate.handleBytesArray,
+        adminStorageAccount: adminStorageKeypair.publicKey,
+        bumpSeed: delegate.userBumpSeed,
+        trackMetadata,
+        userAuthorityKeypair: delegate.userAuthorityDelegateKeypair, // substitute delegate
+        trackOwnerPDA: delegate.userAccountPDA,
+        userAuthorityDelegateAccountPDA: delegate.userAuthorityDelegatePDA,
+        authorityDelegationStatusAccountPDA:
+          delegate.authorityDelegationStatusPDA,
+      })
+    ).to.be.rejectedWith(Error);
+
+    // use different user authority delegate PDA
+    const badDelegate = await testCreateUserDelegate({
+      adminKeypair,
+      adminStorageKeypair: adminStorageKeypair,
+      program,
+      provider,
+      ...getURSMParams(),
+    });
+
+    await expect(
+      testCreateTrack({
+        provider,
+        program: anchor.web3.Keypair.generate().publicKey, // wrong program
```

Comment:
you don't need multiple inputs wrong here - just one wrong param per test case right?